pages/add/form/detail-info-fields.js

```diff
@@ -27,30 +27,38 @@ module.exports = {
   related_creators: {
     type: Creators,
     label: `Name any creators, contributors, partners. Comma separated.`,
+    labelClassname: `form-label`,
     fieldClassname: `form-control`
   },
   interest: {
     type: `text`,
     label: `Why might this be interesting to other people in our network?`,
     placeholder: ``,
-    fieldClassname: `form-control`,
+    labelClassname: `form-label`,
+    fieldClassname: `form-control mb-4`,
     validator: validator.maxLengthValidator(300)
   },
   issues: {
     type: IssuesField,
     label: <IssuesLabel />,
     colCount: 1
+    
   },
   tags: {
     type: Tags,
     label: `Tags: Comma separated. Spaces are ok. Issues are added automatically.`,
+    labelClassname: `form-label`,
```

Comment:
Why is this so WET? If you’re adding this class to every label, is there another way to do it?

Reply:
(Not a blocker, just...curious)

Reply:
Thing is, some `label`s have another element inside of them. Let me check it out and see if there's some improvements I can make.

Reply:
After checking, I think this is the best approach. The class is only added to specific labels that share these styles. I can't really target these labels without also targeting the checkbox labels.